build/images/Dockerfile.simulator.build.ubuntu

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.15 as antrea-build
+
+WORKDIR /antrea
+
+COPY go.mod /antrea/go.mod
+
+RUN go mod download
+
+COPY . /antrea
+
+RUN make antrea-agent-simulator
+
+
+FROM antrea/base-ubuntu:2.14.0
```

Comment:
does the simulator actually depend on this image? does it need Open vSwitch?
if not, it's better to just use ubuntu:20.04 as the base here; it will lead to a smaller image and will be easier to maintain IMO

Reply:
thanks, it does not need ovs, will change to ubuntu:20.04